buildenv/jenkins/perfPipeline.groovy

```diff
@@ -156,7 +101,7 @@ def triggerJob(benchmarkName, platformName, buildParams, jobSuffix) {
 def generateChildJobViaAutoGen(newJobName) {
     def jobParams = []
     jobParams << string(name: 'TEST_JOB_NAME', value: newJobName)
-    jobParams << string(name: 'ARCH_OS_LIST', value: params.PLATFORM)
+    jobParams << string(name: 'PLATFORM', value: params.PLATFORM)
```

Comment:
I believe we need `ARCH_OS_LIST`. See 
 https://github.com/adoptium/aqa-tests/blob/e1538efcebb6eab3c2cdfdd7a042317cb78652d2/buildenv/jenkins/testJobTemplate#L105